chat-commands.js

```diff
@@ -2503,47 +2519,81 @@ exports.commands = {
 			} else if (target === 'dnsbl' || target === 'datacenters') {
 				Dnsbl.loadDatacenters();
-				return this.sendReply("Dnsbl has been hotpatched.");
-			} else if (target.startsWith('disablechat')) {
-				if (Monitor.hotpatchLockChat) return this.errorReply("Hotpatch is already disabled.");
-				let reason = target.split(', ')[1];
-				if (!reason) return this.errorReply("Usage: /hotpatch disablechat, [reason]");
-				Monitor.hotpatchLockChat = reason;
-				return this.sendReply("You have disabled hotpatch until the next server restart.");
+				this.sendReply("Dnsbl has been hot-patched.");
 			} else if (target.startsWith('disable')) {
-				let reason = target.split(', ')[1];
-				if (!reason) return this.errorReply("Usage: /hotpatch disable, [reason]");
-				Monitor.hotpatchLock = reason;
-				return this.sendReply("You have disabled hotpatch until the next server restart.");
+				this.sendReply("Disabling hot-patch has been moved to its own command:");
+				return this.parse('/help nohotpatch');
+			} else {
+				return this.errorReply("Your hot-patch command was unrecognized.");
 			}
 		} catch (e) {
-			return this.errorReply("Something failed while trying to hotpatch " + target + ": \n" + e.stack);
+			Rooms.global.notifyRooms(['development', 'staff', 'upperstaff'], `|c|${user.getIdentity()}|/log ${user.name} used /hotpatch ${target} - but something failed while trying to hot-patch.`);
+			return this.errorReply(`Something failed while trying to hot-patch ${target}: \n${e.stack}`);
 		}
-		this.errorReply("Your hot-patch command was unrecognized.");
+		Rooms.global.notifyRooms(['development', 'staff', 'upperstaff'], `|c|${user.getIdentity()}|/log ${user.name} used /hotpatch ${target}`);
 	},
-	hotpatchhelp: ["Hot-patching the game engine allows you to update parts of Showdown without interrupting currently-running battles. Requires: ~",
+	hotpatchhelp: [
+		"Hot-patching the game engine allows you to update parts of Showdown without interrupting currently-running battles. Requires: ~",
 		"Hot-patching has greater memory requirements than restarting.",
 		"/hotpatch chat - reload chat-commands.js and the chat-plugins",
 		"/hotpatch battles - spawn new simulator processes",
 		"/hotpatch validator - spawn new team validator processes",
 		"/hotpatch formats - reload the sim/dex.js tree, rebuild and rebroad the formats list, and spawn new simulator and team validator processes",
 		"/hotpatch dnsbl - reloads Dnsbl datacenters",
-		"/hotpatch disable, [reason] - disables the use of hotpatch until the next server restart"],
+		"/hotpatch punishments - reloads new punishments code",
+		"/hotpatch tournaments - reloads new tournaments code",
+		"/hotpatch all - hot-patches chat, tournaments, formats, login server, punishments, and dnsbl",
+	],
+
+	hotpatchlock: 'nohotpatch',
+	disablehotpatch: 'nohotpatch',
+	hotpatchdisable: 'nohotpatch',
+	nohotpatch: function (target, room, user) {
```

Comment:
Yeah, I don't feel strongly either way - I just don't see how this being a dev command should have to do with a few less aliases is all